### DISCUSSION TITLE
Allow JSON output format for GitHub Issue templates

### DISCUSSION THREAD

**Discussion by xhluca** [community/community#40222] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Right now, issue templates are rendered directly into markdown, where each section title becomes an H3 header. This format is human readable, but makes automation trickier, as we would need to handle scenarios where markdown is used inside a form field. 

Having an option to access a filled issue form as a JSON would simplify automation and I believe this could increase adoption.

**Comment by ossanna16** [comment]
Hi there @xhluca and welcome to our community! Thank you for the feedback!

**Comment by MarcusAuberg** [comment]
For now we are using a custom script to parse the markdown into json, so this would be a great feature for easier automation
